src/lambda/handler-runner/docker-runner/DockerContainer.js

```diff
@@ -125,18 +156,28 @@ export default class DockerContainer {
   }
 
   async stop() {
-    if (this.#containerId) {
+    if (await this.isRunning()) {
       try {
-        await execa('docker', ['stop', this.#containerId])
-        await execa('docker', ['rm', this.#containerId])
+        await this.#dockerClient.getContainer(this.#containerId).kill()
```

Comment:
Please add `remove()` after `kill()`. `kill()` does not remove stopped container.